public/app/plugins/datasource/testdata/partials/query.editor.html

```diff
@@ -116,4 +116,8 @@
 			<gf-form-switch class="gf-form" label="Level" label-class="query-keyword width-5" checked="ctrl.target.levelColumn" switch-class="max-width-6" on-change="ctrl.refresh()"></gf-form-switch>
 		</div>
 	</div>
+
+	<div>
+		<p ng-bind="ctrl.scenario.description"></p>
```

Comment:
Not sure if this should have any classes assigned to it.

Reply:
I don't think this text help anything -- can we remove it?